lib/cog/bundle/install.ex

```diff
@@ -15,7 +15,7 @@ defmodule Cog.Bundle.Install do
 
   require Logger
 
-  @command_attrs ["name", "version", "options", "enforcing",
+  @command_attrs ["name", "options", "enforcing",
```

Comment:
```elixir
iex(1)> Cog.Models.Command.__schema__(:fields)
[:id, :name, :documentation, :enforcing, :calling_convention, :execution, :bundle_id]
iex(2)> Cog.Models.Command.__schema__(:associations)
[:bundle, :rules, :options]
```
We'd need to filter out the two IDs from the fields, and know which of the associations we care about. Seems like we'd need some additional annotations in order to do this more generally. Maybe we can leave this as is for now; if there are other instances where this pattern pops up we could reconsider then.